go.mod

```diff
@@ -0,0 +1,14 @@
+module github.com/maiqueb/fosdem-2023-cni-chaining
```

Comment:
we may want to consider vendoring this, just in case there's no network

Reply:
good point!

Reply:
Done. Thanks for raising.